### ISSUE TITLE
HTML imports block pre-rendering in Chrome

### ISSUE THREAD

**Issue by Artur-** [vaadin/flow#18]
HTML imports by design blocks rendering (paint) of the page. For apps which can be fully pre-rendered, e.g. TodoMVC, this delays showing of the UI to the user until all HTML imports have completed (not scripts inside HTML imports, these are considered deferred). Today this is a problem only in Chrome which has native support for HTML imports. Other browsers need the webcomponentsjs polyfill before starting imports, and the polyfill is loaded deferred so there is no way that HTML imports can block rendering
